source/Command.h

```diff
@@ -122,21 +122,19 @@ class Command {
 	
 	
 private:
-	explicit Command(uint64_t state);
-	Command(uint64_t state, const std::string &text);
+	explicit Command(uint32_t state);
+	Command(uint32_t state, const std::string &text);
 	
 	
 private:
-	// The key commands and weapons to fire are stored in a single bitmask, with
```

Comment:
Is the reordering intentional?

Reply:
Was trying to get a bit of speed from trying to keep the big data block in cache, however, after profiling, it looks like firing_weps might actually need that more; by putting `aim` last, it hopefully makes firing_weps is faster to access.

Additionally, making `aim` the 'dangling' variable is a matter of risk mitigation: at the moment, ships generally only inhabit the lower reaches of the `aim` array, making high-address cache misses less likely.